README.md

```diff
@@ -72,17 +72,24 @@ options:
                         .duct/logs/{datetime_filesafe}-{pid}_)
   --summary-format SUMMARY_FORMAT
                         Output template to use when printing the summary
-                        following execution. (default: Exit Code: {exit_code}
-                        Command: {command} Log files location: {logs_prefix}
-                        Wall Clock Time: {wall_clock_time:.3f} sec Memory Peak
-                        Usage (RSS): {peak_rss} bytes Memory Average Usage
-                        (RSS): {average_rss} bytes Virtual Memory Peak Usage
-                        (VSZ): {peak_vsz} bytes Virtual Memory Average Usage
-                        (VSZ): {average_vsz} bytes Memory Peak Percentage:
-                        {peak_pmem}% Memory Average Percentage:
-                        {average_pmem}% CPU Peak Usage: {peak_pcpu}% Average
-                        CPU Usage: {average_pcpu}% Samples Collected:
-                        {num_samples} Reports Written: {num_reports} )
+                        following execution. Accepts custom conversion flags:
+                        !S: Converts filesizes to human readable units, green
+                        if measured, red if None. !E: Colors exit code, green
+                        if falsey, red if truthy, and red if None. !X: Colors
+                        green if truthy, red if falsey. !N: Colors green if
+                        not None, red if None (default: Summary: Exit Code:
+                        {exit_code!E} Command: {command} Log files location:
+                        {logs_prefix} Wall Clock Time: {wall_clock_time:.3f}
+                        sec Memory Peak Usage (RSS): {peak_rss!S} Memory
+                        Average Usage (RSS): {average_rss!S} Virtual Memory
+                        Peak Usage (VSZ): {peak_vsz!S} Virtual Memory Average
+                        Usage (VSZ): {average_vsz!S} Memory Peak Percentage:
+                        {peak_pmem!N}% Memory Average Percentage:
+                        {average_pmem!N}% CPU Peak Usage: {peak_pcpu!N}%
+                        Average CPU Usage: {average_pcpu!N}% Samples
+                        Collected: {num_samples!X} Reports Written:
+                        {num_reports!X} )
+  --colors              Use colors in duct output. (default: False)
```

Comment:
in general flags are "suboptimal" since sooner or later boolean becomes insufficient. E.g. here I think default could have been `"auto"` (per initial draft of this PR -- whenever all in/outs are tty) and then we could have `"on", "off"` for explicit values.

Tricky part here would be though if we keep using logger for the summary since in file we might not want those colors etc... 

for now let's just keep this in mind but I think we might reconsider

Reply:
+1 I made default off for "safety first" reasons to avoid adding that complexity in this PR.  

 https://github.com/con/duct/issues/189